app/services/scrapers/high_priority.rb

```diff
@@ -15,10 +15,13 @@ def self.perform(county_name, days_ago: 7, days_forward: 7)
 
     def perform
       cases = fetch_case_list
-
       bar = ProgressBar.new(cases.count)
       puts "#{cases.count} are high priority for update for #{county.name} county"
       cases.each do |case_number|
+        court_case = ::CourtCase.find_by!(county_id: @county.id, case_number: case_number)
+        next unless court_case.enqueued == false
+
+        court_case.update(enqueued: true)
```

Comment:
this is one way to solve it but it would be more performant if you switch from doing the database query in maybe the fetch_case_list method to retrieve all of them at once